edk2toolext/invocables/edk2_setup.py

```diff
@@ -177,8 +177,11 @@ def Go(self):
                         if self.omnicache_path is not None:
                             cmd_string += " --reference " + self.omnicache_path
                         cmd_string += " " + required_repo
-                        RunCmd('git', cmd_string, workingdir=workspace_path,
-                               logging_level=logging.DEBUG, raise_exception_on_nonzero=True)
+                        ret = RunCmd('git', cmd_string, workingdir=workspace_path,
+                                     logging_level=logging.DEBUG, raise_exception_on_nonzero=False)
+                        if ret != 0:
+                            logging.error("We failed to fetch " + required_repo)
+                            raise ValueError("Unable to checkout repo due to error")
```

Comment:
We should change this to RuntimeError so that it's caught by the except and doesn't create an alternate error path.

Reply:
Agreed. Fixing.